Gordon360/Services/LostAndFoundService.cs

```diff
@@ -94,6 +95,44 @@ public int CreateMissingItemReport(MissingItemReportViewModel reportDetails, str
             return reportID;
         }
 
+        /// <param name="id">The id</param>
+        public int CreateActionTaken(int id, ActionsTakenViewModel ActionsTaken)
+        {
+            var account = context.ACCOUNT.FirstOrDefault(x => x.AD_Username == ActionsTaken.username);
+
+            string idNum;
+
+            if (account != null)
+            {
+                idNum = account.gordon_id;
+            }
+            else
+            {
+                throw new ResourceCreationException() { ExceptionMessage = "No account could be found for the user." };
+            }
+
+            var newActionTaken = context.ActionsTaken.Add(new ActionsTaken
+
```

Comment:
Blank line here is a bit confusing, should probably be removed.  (not a big deal, just minor style thing)